src/modules/components/proposal/proposalActions/actions/proposalActionWithdrawToken/proposalActionWithdrawToken.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { render, screen } from '@testing-library/react';
+import { AssetTransfer } from '../../../../asset';
+import type { IProposalActionWithdrawToken } from '../../proposalActionsTypes';
+import { generateProposalActionWithdrawToken } from '../generators/proposalActionWithdrawToken';
+import { type IProposalActionWithdrawTokenProps, ProposalActionWithdrawToken } from './proposalActionWithdrawToken';
+
+jest.mock('../../../../asset', () => ({
+    AssetTransfer: jest.fn(() => <div>Mock AssetTransfer</div>),
+}));
+
+describe('<ProposalActionWithdrawToken /> component', () => {
+    const createTestComponent = (props?: Partial<IProposalActionWithdrawToken>) => {
+        const defaultProps: IProposalActionWithdrawTokenProps = {
+            action: generateProposalActionWithdrawToken(props),
+        };
+
+        return <ProposalActionWithdrawToken {...defaultProps} />;
+    };
+
+    it('renders without crashing', () => {
```

Comment:
The `renders without crashing` name is very generic, here you're testing that it renders the AssetTransfer component
```suggestion
    it('renders the AssetTransfer component', () => {
```